tools/cli/commands/deleteStack.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs';
+import chalk from 'chalk';
+import deleteStack from '../helpers/deleteStack';
+
+import { STACK_LIST, BASE_PATH } from '../config';
+
+const handleDeleteStackList = (stackList, logger, isShowStackList) => {
+  const existsStackList = fs
+    .readdirSync(`${BASE_PATH}/packages`)
+    .filter(stack => stack !== 'common' && stack !== 'mobile')
+    .map(stack => (stack === 'client' ? 'react' : STACK_LIST[stack]));
+
+  if (isShowStackList) {
+    logger.info(chalk.yellow(`List exists stack of technology: ${existsStackList.join(', ')}`));
```

Comment:
"List exists stack of technology" should be renamed to "Existing tech stack list"

Reply:
done